test/stdlib/Inputs/VariadicBool/variadicBool.h

```diff
@@ -0,0 +1,5 @@
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+extern int numberOfTrues(int count, va_list arguments);
```

Comment:
I don't know much about C, but shouldn't there be a calling convention specified, or does it default to "C"?

Reply:
I believe the `-x c` in the run line is enough to get it to C